examples/cosmos/ics20/ics20.qnt

```diff
@@ -45,10 +45,12 @@ module ics20 {
   type ChainState = {
     bank: Accounts,
     channels: ChannelCounterparties,
-    outPackets: Set[Packet],
-    receivedButUnacknowledgedPackets: Set[Packet],
-    inAcknowledgements: Set[FungibleTokenPacketAcknowledgement],
     channelEscrowAddresses: Channel -> Address,
+    // IBC packet handling:
+    outPackets: Set[Packet],                        // outgoing unreceived packets
+    receivedButUnacknowledgedPackets: Set[Packet],  // received but unacknowledged packets
```

Comment:
how about `rcvdToAckPackets`? I am just worried about very long names, which are hard to type.

Also, is this set on the sender side or the receiver side? Kind of a silly question, but it would be nice to understand.

Reply:
See https://github.com/informalsystems/quint/pull/865#discussion_r1187325862. The distinguishing feature is receival, not acknowledgement.

> I am just worried about very long names, which are hard to type.
Why? 🙃